lib/actions/device.coffee

```diff
@@ -186,6 +186,70 @@ exports.reboot =
 		resin = require('resin-sdk')
 		resin.models.device.reboot(params.uuid).nodeify(done)
 
+exports.enableDeviceUrl =
+	signature: 'device enableDeviceUrl <uuid>'
```

Comment:
If we do change the command name, I think it's important to convey it's _public_. In the Dashboard and [docs](http://docs.resin.io/management/devices/#enable-public-device-url) we consistently call the feature "Public Device URL", which could probably map to `device public-url enable` and `device public-url disable` nicely (I'm unfamiliar with CLI's naming conventions but I think you get the picture).